apps/console/src/features/identity-providers/components/wizards/hypr/hypr-authentication-provider-create-wizard.tsx

```diff
@@ -0,0 +1,455 @@
+                            <WizardHelp/>
+                        </Suspense>
+                    </ModalWithSidePanel.Content>
+                </ModalWithSidePanel.SidePanel>
+            );
+        };
+
+        /**
+        * Closure to submit form.
+        */
+        let submitForm: () => void;
+
+        /**
+        * Closure to trigger previous form.
+        */
+        let triggerPreviousForm: () => void;
+
+        return (
+            <>
+                { openLimitReachedModal &&
+                    (
+                        <TierLimitReachErrorModal
+                            actionLabel={ t(
+                                "console:develop.features.idp.notifications." +
+                            "tierLimitReachedError.emptyPlaceholder.action"
+                            ) }
+                            handleModalClose={ handleLimitReachedModalClose }
+                            header={ t(
+                                "console:develop.features.idp.notifications.tierLimitReachedError.heading"
+                            ) }
+                            description={ t(
+                                "console:develop.features.idp.notifications." +
+                            "tierLimitReachedError.emptyPlaceholder.subtitles"
+                            ) }
+                            message={ t(
+                                "console:develop.features.idp.notifications." + 
+                            "tierLimitReachedError.emptyPlaceholder.title"
+                            ) }
+                            openModal={ openLimitReachedModal }
+                        />
+                    ) 
+                }
+                <ModalWithSidePanel
+                    open={ !openLimitReachedModal }
+                    className="wizard identity-provider-create-wizard"
+                    dimmer="blurring"
+                    onClose={ handleWizardClose }
+                    closeOnDimmerClick={ false }
+                    closeOnEscape
+                    data-testid={ `${ testId }-modal` }
+                >
+                    <ModalWithSidePanel.MainPanel>
+                        <ModalWithSidePanel.Header
+                            className="wizard-header"
+                            data-testid={ `${ testId }-modal-header` }
+                        >
+                            <div className="display-flex">
+                                <GenericIcon
+                                    icon={ getIdPIcons().hypr }
+                                    size="mini"
+                                    transparent
+                                    spaced="right"
+                                    data-testid={ `${ testId }-image` }
+                                />
+                                <div className="ml-1">
+                                    { title }
+                                    { subTitle &&
+                                (
```

Comment:
Shall we check formatting here

Reply:
addressed with [f58871f](https://github.com/wso2/identity-apps/pull/3790/commits/f58871f9724393ba68ae73f009cf80d4cd28512f)